Dockerfile

```diff
@@ -24,9 +24,13 @@ RUN jq '. +{"dependencies": (.dependencies + {"dd-trace": .devDependencies."dd-t
 # Copy the production dependencies to the modules folder
 RUN yarn install --production=true
 RUN cp -rf node_modules/* /nodejs/node_modules
+
 # Remove the AWS SDK, which is installed in the lambda by default
 RUN rm -rf /nodejs/node_modules/aws-sdk
 RUN rm -rf /nodejs/node_modules/aws-xray-sdk-core/node_modules/aws-sdk
+
 # Remove heavy files from dd-trace which aren't used in a lambda environment
 RUN rm -rf /nodejs/node_modules/dd-trace/prebuilds
 RUN rm -rf /nodejs/node_modules/dd-trace/dist
+RUN rm -rf /nodejs/node_modules/hdr-histogram-js/build
+RUN rm -rf /nodejs/node_modules/protobufjs/dist
```

Comment:
Good catch on the protobufjs/dist folder. Looks like it's just used for importing from the frontend when you don't have a webpacking build step.